common/models/role.js

```diff
@@ -240,6 +241,15 @@ module.exports = function(Role) {
     Role.isAuthenticated(context, callback);
   });
 
+  Role.registerResolver(Role.EVERYONE_PROMISE, function(role, context) {
```

Comment:
I think you should not create a named constant like `Role.EVERYONE_PROMISE`, and simply use an anonymous string in the tests.

```js
Role.registerResolver('TEST_PROMISE_EVERYONE', /*...*/);
Role.isInRole('TEST_PROMISE_EVERYONE', /*...*/);
```